librz/core/cmd/cmd_analysis.c

```diff
@@ -7061,6 +6811,127 @@ RZ_IPI RzCmdStatus rz_analysis_function_cc_reg_usage_handler(RzCore *core, int a
 	return RZ_CMD_STATUS_OK;
 }
 
+RZ_IPI RzCmdStatus rz_analysis_function_describe_offset_handler(RzCore *core, int argc, const char **argv) {
+	RzList *list = rz_analysis_get_functions_in(core->analysis, core->offset);
+	if (rz_list_empty(list)) {
+		RZ_LOG_ERROR("No function found in 0x%08" PFMT64x ".\n", core->offset);
+		rz_list_free(list);
+		return RZ_CMD_STATUS_ERROR;
+	}
+	RzListIter *it;
+	RzAnalysisFunction *fcn;
+	rz_list_foreach (list, it, fcn) {
+		st64 delta = core->offset - fcn->addr;
+		if (delta > 0) {
+			rz_cons_printf("%s + %" PFMT64d "\n", fcn->name, delta);
+		} else if (delta < 0) {
+			rz_cons_printf("%s - %" PFMT64d "\n", fcn->name, -delta);
+		} else {
+			rz_cons_printf("%s\n", fcn->name);
+		}
+	}
+	rz_list_free(list);
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_analysis_function_add_recu_handler(RzCore *core, int argc, const char **argv) {
+	const char *name = argc == 2 ? argv[1] : NULL;
+	bool analyze_recursively = true;
+	if (!strcmp(argv[0], "af")) {
```

Comment:
can this be called from anywhere else?

Reply:
Yes, `afr`

Reply:
Btw I don't see a difference between `af` and `afr`. This `analysis.call` flag is really confusing. `af` internals anyway invoke `rz_analysis_analyze_fcn_refs`.

Reply:
i suggest to check it via `git blame` often it helps